packages/envd/internal/host/metrics.go

```diff
@@ -9,9 +9,9 @@ import (
 )
 
 type Metrics struct {
-	CPU       float64 `json:"cpu_pct"`   // Percent rounded to 2 decimal places
-	Mem       uint64  `json:"mem_bytes"` // Total virtual memory in bytes
-	Timestamp int64   `json:"ts"`        // Unix Timestamp in UTC
+	CPUPercent float64 `json:"cpu_pct"` // Percent rounded to 2 decimal places
+	MemMB      uint64  `json:"mem_mb"`  // Total virtual memory in MB
```

Comment:
Open to discussion, but I think it would be nice to start using MiB to prevent any confusion.

Reply:
per discussion: makes sense to use MiB in this case since we're already using powers of 2 — eg `x / 1024  / 1024`